syntax = "proto3";

message RpcHeader {
    string service_name = 1;
    string method_name = 2;
    uint32 args_size = 3;
}